tests/resources/test_resources.py

```diff
@@ -102,7 +103,7 @@ def test_create_banner(client, admin, headers):
     assert banner["message"] == banner_data["message"]
     assert banner["url_path"] == banner_data["url_path"]
     assert banner["category"] == banner_data["category"]
-    assert banner["active"] == banner_data["active"]
+    assert banner["active"] != banner_data["active"]
```

Comment:
not sure why this has been worked the last test runs. this change should be obvious because the enddate of the banner is in the past. active is set to false due to https://github.com/inveniosoftware/invenio-banners/blob/24a96abdbcf57bc0cc02469693505f9fe3cd8fac/invenio_banners/resources/resource.py#L93. this should have been also the case the last run in february, don't know why it pops up now